### DISCUSSION TITLE
Changing the hover color for Buttons

### DISCUSSION THREAD

**Discussion by Amogh-Bharadwaj** [primer/react#2107] [Q&A] [answered]
Is there a way to change hover and selected properties of a Button?
I've tried the sx prop as:
`<Button sx={{':hover':{background:'neutral.muted'}}} variant='invisible'>Hello World</Button>`
But the hover colour still stays white.
Removing the variant attribute or changing it to other variants yields the same.
Any help would be appreciated, thanks.

**Comment by issammani** [comment, marked as answer]
Hi @Amogh-Bharadwaj, 

This is a specificity issue. This is actually very common with component libraries that allow overriding styles like this. You can use the `&&` operator to increase the specificity of your overriding hover styles like so: 
```jsx 
<Button sx={{'&&:hover':{background:'neutral.emphasis'}}}>Hello World</Button>
```

Here is a working example in [codesandbox](https://codesandbox.io/s/floral-hill-wwt273?file=/src/App.tsx). 

PS: I changed the color from `neutral.muted` to `neutral.emphasis` so you can see the difference. The default hover color and `neutral.muted` are very similar.

**Reply by Amogh-Bharadwaj** [reply]
Thank you so much!
